pyproject.toml

```
@@ -70,7 +70,7 @@ disallow_untyped_defs = false
 no_implicit_optional = false
 show_error_codes = true
 warn_return_any = false
-warn_unused_ignores = true
+warn_unused_ignores = false
```

Let's revert this change and fix any related errors

```suggestion
warn_unused_ignores = true
```